### ISSUE TITLE
Hide the battery block when the device could not be found

### ISSUE THREAD

**Issue by c02y** [greshake/i3status-rust#1668]
I play video game using Xbox One Series game controller with the wireless dongle, so I use battery block to display the batter level of the game controller, when I'm not playing the game, the game controller will be disconnected, but when I reload i3status-rust bar, it displays 
```
UPower device could not be found
```

So could you please hide the battery block when the device could not be found ?

**Comment by MaxVerevkin** [comment]
This is a known limitation of current `driver = "upower"` implementation (#1495). Can you try sysfs driver?

**Comment by c02y** [comment]
The battery info of Xbox game controller of xone driver is fetched by UPower.

Is this implementation planned or impossible?

**Comment by MaxVerevkin** [comment]
> The battery info of Xbox game controller of xone driver is fetched by UPower.

But it's not exposed by sysfs? I wonder where upower fetches the information from.

> Is this implementation planned or impossible?

It's possible but I doubt anyone is working on it.

**Comment by c02y** [comment]
> > The battery info of Xbox game controller of xone driver is fetched by UPower.
> 
> But it's not exposed by sysfs? I wonder where upower fetches the information from.

I'm not familiar with the drivers ("sysfs", "apc_ups", or "upower"), but xone repo says `Battery reporting (UPower integration)`, so I use `UPower`, I tried battery block for the first time and only a few hours.

I tried `sysfs` driver just now, and sysfs's value and upower's value are different, `sysfs` is 75%, `upower` 70%, I don't know which one is more accurate, but `hide_block_when_device_is_not_found` is working for `sysfs` driver.

I need more time, maybe a few days, about the accurate and will let you know the result.
